src/main/java/seedu/address/model/country/CountryManager.java

```diff
@@ -1,29 +1,30 @@
 package seedu.address.model.country;
 
-import java.util.Collection;
-import java.util.Collections;
+import java.util.ArrayList;
 import java.util.LinkedHashMap;
 import java.util.LinkedHashSet;
+import java.util.List;
 import java.util.Locale;
 import java.util.Map;
 import java.util.Set;
 
-import seedu.address.model.note.Note;
+import seedu.address.model.note.CountryNote;
 
 /**
- * A high-level class responsible for mapping ISO3166 2-letter country codes to countries.
+ * A high-level class responsible for mapping ISO3166 countries to countryNotes.
  */
 public class CountryManager {
 
-    //TODO: Should include checking for 3-letter Country Code?
     private static final String[] COUNTRY_CODES = Locale.getISOCountries();
-    private final Map<Country, Set<Note>> countryNotesMap;
+    private final Map<Country, Set<CountryNote>> countryToCountryNoteMap;
```

Comment:
this is really nit-picky of me, but I suggest naming it `countryToCountryNotesMap` to reflect the plurality of the country notes in the set.
```suggestion
    private final Map<Country, Set<CountryNote>> countryToCountryNotesMap;
```

Reply:
Done